model/src/main/java/com/stezhka/entity/MovieCountry.java

```diff
@@ -0,0 +1,27 @@
+package com.stezhka.entity;
```

Comment:
looks like technical entity - can be stored in different place

Reply:
Done